src/app/pages/home/home.page.ts

```diff
@@ -45,6 +45,7 @@ export class HomePage implements OnInit {
         document.getElementById('demo').innerHTML = '&nbsp;&nbsp;&nbsp;EXPLODING NOW !!!';
         // Commence explosions!
         const { cancel } = emojisplosions();
+        this.cancel = cancel;
```

Comment:
I haven't played with this locally so maybe `this` isn't bound to the right thing here. You'll have to debug a bit if it doesn't work like this. :smile: